lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -205,7 +202,7 @@ def saturation_vapour_pressure_goff_gratch(temperature):
         triple_pt = consts.TRIPLE_PT_WATER
 
         # Values for which method is considered valid (see reference).
-        WetBulbTemperature.check_range(temperature, 173., 373.)
+        WetBulbTemperature.check_range(temperature.data, 173., 373.)
```

Comment:
You seem to be assuming temperature will be an array but the description still says it is a cube.

This function also says it will convert to Kelvin prior to calculation but as far as I can see it just assumes Kelvin. This is a problem in the original function. We could change the description to the data will be in Kelvin but again need to ensure all calls will be in the right units.  

I think you want to add a wrapper function to return the data as a cube so that the underlying function could be called just using numpy arrays.

Reply:
This one change in the function so far was because check_range was being used elsewhere.